### DISCUSSION TITLE
Icon don't change color for switch

### DISCUSSION THREAD

**Discussion by lennart-bystrom** [dwainscheeren/dwains-lovelace-dashboard#653] [Q&A]
HI, just installed Dwains, looks really good but the icons for switch does not change color as it does for lights?! What's wrong?

![Skärmbild 2022-11-13 072429](https://user-images.githubusercontent.com/59575811/201508848-90ff6844-0b53-4a7e-bfcd-bc03d4e69680.jpg)

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
